File: WiTrack/app/src/main/java/util/Frame.java
package util;

import android.widget.ArrayAdapter;

import java.util.ArrayList;

/**
 * Created by j388923r on 1/20/16.
 */
public class Frame {

    ArrayList<Position> people;
    String timeStamp;

    public Frame(String time) {
        this.timeStamp = time;
    }

    public Frame(ArrayList<Position> positions, String time) {
        this.people = positions;
        this.timeStamp = time;
    }

    public void addPerson(Position person) {
        people.add(person);
    }

    public ArrayList<Position> getPeople() {
        return people;
    }

    public String getTimeStamp() {
        return timeStamp;
    }
}

File: WiTrackApp/WiTrackApp/Frame.swift
//
//  Frame.swift
//  WiTrackApp
//
//  Created by Jamar Brooks on 1/15/16.
//  Copyright © 2016 Jamar. All rights reserved.
//

import Foundation

class Frame {
    var people = [Position]()
    var timeStamp : String?
    
    init(people: [Position], time : String){
        self.people = people
        self.timeStamp = time
    }
}